lib/web/sessions.go

```diff
@@ -29,8 +29,9 @@ import (
 	"golang.org/x/crypto/ssh"
 	"golang.org/x/crypto/ssh/agent"
 
+	apiclient "github.com/gravitational/teleport/api/client"
```

Comment:
```go
"github.com/gravitational/teleport/api/client"
```

Reply:
"github.com/gravitational/teleport/lib/client" is also imported